package.json

```diff
@@ -16,6 +16,7 @@
   },
   "version": "0.1.0",
   "devDependencies": {
+    "@webpack-cli/serve": "^1.6.1",
```

Comment:
This is for development purposes, remove if you dont use the webpack CLI to host webpack in a separate terminal window. I find it easier